metis-common/src/main/java/eu/europeana/metis/RestEndpoints.java

```diff
@@ -29,6 +29,7 @@
 
   // DEPUBLISHED RECORDS
   public static final String DEPUBLISH_RECORDIDS_DATASETID = "/depublish/record_ids/{datasetId}";
+  public static final String DEPUBLISH_RECORDIDS_EXECUTE_DATASETID = "/depublish/record_ids/execute/{datasetId}";
```

Comment:
Just a small issue perhaps, but I don't think that this path is appropriate, given that the task being executed may not be a depublication for individual records. What about `/depublish/execute/{datasetId}`?

Reply:
Good point.